modeling/cross_validate_runner.py

```diff
@@ -0,0 +1,96 @@
+from dataclasses import dataclass
+from typing import Dict, List, Any, Optional
```

Comment:
forgot about this, why we import these from typing instead of using the python default data structures again?

Reply:
these are only used for mypy checks haha we didn't instantiate any of these classes